docs/src/test-runners-python.md

```diff
@@ -48,6 +48,18 @@ If you want to add the CLI arguments automatically without specifying them, you
 addopts = --headed --browser firefox
 ```
 
+## CLI arguments
```

Comment:
I'd add some links:
- browsers.md  for browser-channel
- emulation.md for device
- tracing.md for tracing